models/intel/action-recognition-0001/composite-model.yml

```
@@ -1 +1,6 @@
+description: >-
+  General-purpose action recognition model for Kinetics-400 dataset based on Video
+  Transformer Network approach. Encoder par
```

looks like description is broken
```suggestion
  Transformer Network approach.
```

---

Done